klee/lib/Core/Executor.cpp

```diff
@@ -450,6 +450,36 @@ Executor::StatePair Executor::fork(ExecutionState &current, const ref<Expr> &con
     return StatePair(trueState, falseState);
 }
 
+Executor::StatePair Executor::fork(ExecutionState &current) {
+    if (current.forkDisabled) {
+        return StatePair(&current, nullptr);
+    }
+
+    ExecutionState *clonedState;
+    notifyBranch(current);
+    clonedState = current.clone();
+    addedStates.insert(clonedState);
+
+    // Recompute concrete values.
+    ArrayVec symbObjects = current.symbolics;
+    std::vector<std::vector<unsigned char>> concreteObjects;
+
+    Query q(current.constraints(), ConstantExpr::alloc(0, Expr::Bool));
```

Comment:
You don't need to recompute concolic values in the cloned state. They should stay the same.

Reply:
It's working as expected now! Thanks so much.